integrations/docker/images/chip-build/Dockerfile

```diff
@@ -60,6 +62,19 @@ RUN set -x \
     && rm cmake-3.17.2.tar.gz \
     && : # last line
 
+# Install newer git version, to allow for github workflow checkout
+RUN set -x \
```

Comment:
I was just updating us to ubuntu:focal, which already has this and an up-to-date cmake

Reply:
Is that change already in?

I have already built the 0.2.18 image in dockerhub to unblock linux with libdbus-1-dev availability. I am afraid that if we keep the PRs the images and sources become out of sync.